alpakaConfig.cmake

```diff
@@ -357,6 +379,46 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 ENDIF()
 
             ELSE()
+                IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+                    IF(CUDA_VERSION VERSION_EQUAL 8.0)
+                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.0)
```

Comment:
I was not aware that we have the gcc guard also added to alpaka and not only to PIConGPU.
GCC 5.3 and CUDA 8 are compatible (https://github.com/ComputationalRadiationPhysics/picongpu/pull/2636)
I will add a PR to alpaka to fix it.

Reply:
Added in #566